velox/expression/tests/FuzzerRunner.cpp

```diff
@@ -91,10 +91,7 @@ getSignaturesForCast() {
   addCastFromFloatingPointSignatures("varchar", signatures);
   addCastFromVarcharSignature("varchar", signatures);
   addCastFromDateSignature("varchar", signatures);
-
-  // TODO: Enable this signature after we fix the timestamp --> varchar
-  // conversion. https://github.com/facebookincubator/velox/issues/4169.
-  // addCastFromTimestampSignature("varchar", signatures);
+  addCastFromTimestampSignature("varchar", signatures);
```

Comment:
Nice. Let's run the fuzzer for 30+ minutes to make sure there are no failures.

Reply:
Is it going to run automatically for PR or do I need to do it locally? Is there any documentation in how to do it

Reply:
It will run only for 10 min. Then there is a nightly job that runs for 1h. Please, run manually for 30min+ to make sure nightly job doesn't fail.

See https://github.com/facebookincubator/velox/blob/main/CONTRIBUTING.md#prestos-sql-functions

The specific command can be found in .circleci/config.yml